Landing/elements/textCutter/index.tsx

```diff
@@ -0,0 +1,68 @@
+// Core
+import React, {FC, useEffect, useState, useContext} from 'react';
+
+// View
+import * as S from './styles'
+
+// Locale
+import { LocalContext } from '../../app/page'
+
+
+
+interface TextCutterT {
+    children: string
+}
+
+export const TextCutter:FC<TextCutterT>= ({children}) => {
+    const [isOpen, setIsOpen] = useState(false)
+
+    useEffect(()=>{
+        if (children?.length <=400) {
+            setIsOpen(true)
+        }
+    }, [children])
+
+    const { local } = useContext(LocalContext)
+    
+    const ButtonLocale = () => {
+        let btnName 
+
+        switch(local){
+            case 'en': btnName = "read more";
```

Comment:
WE can create a enum with that and use enum here

Reply:
done but i dont get why is it better to use enum